python.html.markdown

```diff
@@ -35,22 +35,26 @@ allow you to write Python 3 code that will run on Python 2, so check out the Pyt
 ## 1. Primitive Datatypes and Operators
 ####################################################
 
-# You have numbers
+# You have numbers, which are type int
 3  # => 3
+type(3) # => <type 'int'>
 
 # Math is what you would expect
-1 + 1  # => 2
-8 - 1  # => 7
-10 * 2  # => 20
-35 / 5  # => 7
+1 + 1  # addition => 2
```

Comment:
I don't think it's necessary to explain what `+` does to most programmers. Nor the following three lines.